src/raylib/textures/Image.cpp

```diff
@@ -20,13 +20,40 @@ namespace raylib
 
         Image::~Image() { unload(); }
 
-        void Image::load() { _image = LoadImage(_fileName.generic_string().c_str()); }
+        void Image::load()
+        {
+            unload();
+            if (!_fileName.empty())
+                _fileName = " ";
+            _image = LoadImage(_fileName.generic_string().c_str());
+            _loaded = true;
+        }
 
-        void Image::loadFromTexture(const Texture2D &texture) { _image = LoadImageFromTexture(texture); }
+        void Image::loadFromTexture(const Texture2D &texture)
+        {
+            unload();
+            if (!_fileName.empty())
+                _fileName = " ";
```

Comment:
Set an empty string **""**, not a string containing a whitespace (" ".empty() will return false)